docs/source/development/set_up_pycharm.md

```diff
@@ -50,22 +50,16 @@ Add a new **Python** Run configuration:
 
 ![](../meta/images/pycharm_add_py_run_config.png)
 
-Create a **Run / Debug Configuration** for `kedro run` and get the path to the Kedro CLI script:
+Specify the **Run / Debug Configuration** name in the **Name** field, and edit the new Run configuration as follows:
 
-```console
-# macOS / Linux
-which kedro
-
-# Windows (in `cmd`)
-python -c "import sys, os.path; print(os.path.join(os.path.dirname(sys.executable), 'Lib\site-packages\kedro\__main__.py'))"
-```
-
-Edit the new Run configuration as follows:
+- Pick **Module** from the dropdown
```

Comment:
The screenshot shows "script" instead of "module", right? I guess the text is correct and the screenshot needs to be taken again

Reply:
Whoops - was the third time I redid that screen grab and apparently wasn't paying enough attention!